src/packet_analysis/protocol/ethernet/Ethernet.cc

```diff
@@ -10,6 +11,26 @@ EthernetAnalyzer::EthernetAnalyzer()
 	{
 	}
 
+void EthernetAnalyzer::Initialize()
+	{
+	SNAPAnalyzer = LoadAnalyzer("PacketAnalyzer::Ethernet::snap_analyzer");
```

Comment:
Where are these analyzers defined? Something coming later?

Reply:
So far we just ported existing functionality. In case of Ethernet, there are three "sub-protocols" that haven't been supported so far. Because the distinction does use the straight forward ID mechansims, the Ethernet analyzer defines constants for configuration:
```zeek
export {
	## IEEE 802.2 SNAP analyzer
	const snap_analyzer: PacketAnalyzer::Tag &redef;
	## Novell raw IEEE 802.3 analyzer
	const novell_raw_analyzer: PacketAnalyzer::Tag &redef;
	## IEEE 802.2 LLC analyzer
	const llc_analyzer: PacketAnalyzer::Tag &redef;
}
```